lib/portlayer/storage/vsphere/image.go

```diff
@@ -280,6 +280,7 @@ func (v *ImageStore) WriteImage(op trace.Operation, parent *portlayer.Image, ID
 // spec - describes filters on paths found in the data (include, exclude, rebase, strip)
 // data - set to true to include file data in the tar archive, false to include headers only
 func (v *ImageStore) Export(op trace.Operation, store *url.URL, id, ancestor string, spec *archive.FilterSpec, data bool) (io.ReadCloser, error) {
+	isNotPersistentDisk := false
```

Comment:
sure, I will use `persistent = true` as a constant and use that verb above. When I wrote that I realized how funny it looked after the fact :P